Add the 'null' value type to the CheckBox.value description (#2766)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxCheckBox/1 Configuration/value.md

Specifies the UI component state.

---
The check box can be in one of the following states depending on the **value** property's value.

- checked  
 **value** is **true**

- unchecked  
 **value** is **false**

- undetermined  
 **value** is **undefined** or **null**

> Users can press the Space key to change the **value**.

Reviewer: Can we add bold font to "Space" as well? For example:

`Users can press the **Space** key to change the **value**.`

Specifies the UI component state.

---
The check box can be in one of the following states depending on the **value** property's value.

- checked  

>  value is **true**

Reviewer 2: ```suggestion
 **value** is **true**
```

>  value is **false**

Reviewer 2: ```suggestion
 **value** is **false**
```

- undetermined  

>  value is **undefined** or **null**

Reviewer 2: ```suggestion
 **value** is **undefined** or **null**
```

> An end-user can use the **Space** key to change the UI component value.

Reviewer 2: ```suggestion
Users can press the Space key to change the **value**.
```
